src/components/development/MAVLinkInspector.vue

```diff
@@ -84,10 +103,16 @@ const toggleMessageTracking = (type: MAVLinkType): void => {
 const setupMessageListeners = (type: MAVLinkType): void => {
   try {
     mainVehicleStore.listenToIncomingMessages(type, (pack: Package) => {
-      messageValues.value.set(`in:${type}`, pack.message)
+      messageValues.value.set(`in:${type}`, {
+        timestamp: new Date().toISOString(),
```

Comment:
```suggestion
        timestamp: new Date().toISOString().replace("T", " ").slice(0, -1),
```
That seems like it would be more useful for actually reading the values :-)